app_flutter/lib/main.dart

```diff
@@ -15,9 +16,16 @@ import 'state/index.dart';
 import 'widgets/now.dart';
 import 'widgets/state_provider.dart';
 
-void main() {
+void main(List<String> args) {
   final GoogleSignInService authService = GoogleSignInService();
-  final CocoonService cocoonService = CocoonService();
+  bool useProductionService = kReleaseMode;
+  if (args.contains('--use-production-service')) {
```

Comment:
Can you document these in `app_flutter/README.md`?

Reply:
Yes, sure.  I did that, and added a usage message and "--help" option.